src/commands/deploy/deploy.ts

```diff
@@ -0,0 +1,186 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as WebSiteModels from 'azure-arm-website/lib/models';
```

Comment:
Would've preferred a separate PR to move the deploy file. Now I can't easily see the diff